plugin/pkg/auth/authorizer/rbac/bootstrappolicy/testdata/controller-roles.yaml

```diff
@@ -907,6 +907,16 @@ items:
     verbs:
     - delete
     - list
+  - apiGroups:
+    - networking.k8s.io
+    resources:
+    - clustercidrconfigs
+    verbs:
+    - create
```

Comment:
why create/delete ?

Reply:
removed create, delete and watch, which are not required, only needs get and list